oic.r.acl2.raml

```diff
@@ -9,6 +9,16 @@ traits:
       queryParameters:
         if:
           enum: ["oic.if.baseline"]
+  - ace-filtered:
+      usage: Apply this to any method that can filter ACEs
```

Comment:
This wasn't unintentional - as I was thinking it was awkward to refer to ACL entries as ACE2s. In the context of ACL2, there's really only one kind of ACE. So figured this was sufficient for human consumption. But I'm not going to lose sleep over it either way.